src/components/calculator.js

```diff
@@ -1,57 +1,53 @@
-
-          <button onClick={clickHandler} type="button">7</button>
-          <button onClick={clickHandler} type="button">8</button>
-          <button onClick={clickHandler} type="button">9</button>
-          <button onClick={clickHandler} className="orange" type="button">x</button>
-
-          <button onClick={clickHandler} type="button">4</button>
-          <button onClick={clickHandler} type="button">5</button>
-          <button onClick={clickHandler} type="button">6</button>
-          <button onClick={clickHandler} className="orange" type="button">-</button>
-
-          <button onClick={clickHandler} type="button">1</button>
-          <button onClick={clickHandler} type="button">2</button>
-          <button onClick={clickHandler} type="button">3</button>
-          <button onClick={clickHandler} className="orange" type="button">+</button>
-
-          <button onClick={clickHandler} type="button">0</button>
-          <button onClick={clickHandler} type="button">.</button>
-          <button onClick={clickHandler} className="orange" type="button">=</button>
-        </div>
-
+function Calculator(props) {
+  const [state, setState] = useState({
+    total: 0,
+    next: 0,
+    operation: '',
+  });
+
+  const clickHandler = (e) => {
+    setState(calculate(state, e.target.textContent));
+  };
+
+  const { next, total } = state;
+
+  return (
+    <div className="calculator">
+      <h1>{ props.name }</h1>
+      <div className="screen-container">{next || total || 0}</div>
+      <div className="touch-pad">
+        <button onClick={clickHandler} type="button">AC</button>
+        <button onClick={clickHandler} type="button">+/-</button>
+        <button onClick={clickHandler} type="button">%</button>
+        <button onClick={clickHandler} className="orange" type="button">÷</button>
+
+        <button onClick={clickHandler} type="button">7</button>
+        <button onClick={clickHandler} type="button">8</button>
+        <button onClick={clickHandler} type="button">9</button>
+        <button onClick={clickHandler} className="orange" type="button">*</button>
+
+        <button onClick={clickHandler} type="button">4</button>
+        <button onClick={clickHandler} type="button">5</button>
+        <button onClick={clickHandler} type="button">6</button>
+        <button onClick={clickHandler} className="orange" type="button">-</button>
+
+        <button onClick={clickHandler} type="button">1</button>
+        <button onClick={clickHandler} type="button">2</button>
+        <button onClick={clickHandler} type="button">3</button>
+        <button onClick={clickHandler} className="orange" type="button">+</button>
+
+        <button onClick={clickHandler} type="button">0</button>
+        <button onClick={clickHandler} type="button">.</button>
+        <button onClick={clickHandler} className="orange" type="button">=</button>
       </div>
-    );
-  }
+
+    </div>
+  );
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you fix your design and stick to the layout that was provided earlier.

  Your design | Required design
  :---:|:---:
  ![eieoe](https://user-images.githubusercontent.com/36542986/205454612-350d21df-1276-457f-b5e3-b37fbbe439cb.jpg) | ![rorpr](https://user-images.githubusercontent.com/36542986/205454610-a1a2d4db-938b-4b42-9ebf-109d0fabbe1e.jpg)